VERSION.cmake

```diff
@@ -1,11 +1,11 @@
 SET(VERSION_MAJOR "2")
-SET(VERSION_MINOR "49")
-SET(VERSION_PATCH "16")
+SET(VERSION_MINOR "50")
+SET(VERSION_PATCH "0")
 SET( VERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}${GIT_SHA1_VERSION}" )
 
 ##### This is needed for the libyui-qt core ONLY.
 ##### These will be overridden from exports in LibyuiConfig.cmake
-SET( SONAME_MAJOR "5" )
-SET( SONAME_MINOR "1" )
+SET( SONAME_MAJOR "9" )
```

Comment:
This needs to be updated to `10`, see https://github.com/libyui/libyui/pull/141#discussion_r274357536